src/controllers/simplified-account/services/payment-links/delete/delete-payment-link.controller.ts

```diff
@@ -1,17 +1,211 @@
 import { ServiceRequest, ServiceResponse } from '@utils/types/express'
+import { response } from '@utils/response'
+import { getProductByExternalId } from '@services/products.service'
+import { penceToPoundsWithCurrency } from '@utils/currency-formatter'
+import ProductsClient from '@services/clients/pay/ProductsClient.class'
+import formatServiceAndAccountPathsFor from '@utils/simplified-account/format/format-service-and-account-paths-for'
+import paths from '@root/paths'
+import createLogger from '@utils/logger'
 
-function get (req: ServiceRequest, res: ServiceResponse) {
-  return res.status(501).json({
-    message: 'not implemented'
-  })
+const logger = createLogger(__filename)
+const productsClient = new ProductsClient()
+
+interface DeletePaymentLinkBody {
+  'confirm-delete'?: string
+}
+
+interface ValidationError {
+  summary: { href: string; text: string }[]
+}
+
+interface SessionData {
```

Comment:
do we need to use session data to handle form state here? adds a lot of complexity so better to avoid it if not required

Reply:
Removed.